conf/ocsci/ibmcloud_disable_odf_addon_deployment.yaml

```diff
@@ -0,0 +1,4 @@
+---
+# This config file can be used for disable the OCS addon
+DEPLOYMENT:
+  ibmcloud_disable_addon: Falue
```

Comment:
Falue?  

Anyway, in this file this value should be True.

As if you would like to disable addon, you will pass this file and hence ibmcloud_disable_addon should be True.